### DISCUSSION TITLE
寻求帮助：push_attach_task的参数含义

### DISCUSSION THREAD

**Discussion by ZorianLi** [ttttupup/wxhelper#357] [Q&A]
下载附件的API，push_attach_task(mgr, p_chat_msg, 0, 1)，第三个和第四个参数的含义是什么？我尝试了重复下载附件，但是实际结果并没有重复下载，我猜测这两个参数是不是有一个控制是否强制下载的？

**Comment by ttttupup** [comment]
控制立即下载还是加入到队列中下载
